dist/gcc-compatible/Makefile.basic

```diff
@@ -14,7 +14,7 @@ endif
 CFLAGS 	+= -I. -I $(KREMLIN_HOME)/include -I $(KREMLIN_HOME)/kremlib/dist/minimal
 CFLAGS 	+= -Wall -Wextra -Werror -std=c11 -Wno-unused-variable \
   -Wno-unknown-warning-option -Wno-unused-but-set-variable \
-  -Wno-unused-parameter -Wno-infinite-recursion \
+  -Wno-unused-parameter -Wno-infinite-recursion -Wno-unused-function \
```

Comment:
why did you need this one, out of curiosity?

Reply:
`vale_impl_of_alg`, `poly1305_vale`, and `has_adx_bmi2` are `static` functions that aren't used but always get built.